components/BottomNavigation/src/MDCBottomNavigationBarController.m

```diff
@@ -335,9 +347,56 @@ - (void)addContentViewController:(UIViewController *)viewController {
   if (viewController && doesNotContainViewController) {
     [self addChildViewController:viewController];
     [self.content addSubview:viewController.view];
-    [self addConstraintsForContentView:viewController.view];
+    [self addConstraintsForChildViewControllerView:viewController.view];
     [viewController didMoveToParentViewController:self];
   }
+  [self updateNavigationBarInsets];
+}
+
+- (void)viewDidLayoutSubviews {
+  [super viewDidLayoutSubviews];
+
+  [self updateNavigationBarInsets];
+}
+
+- (void)updateNavigationBarInsets {
+  UIEdgeInsets currentSafeAreaInsets = UIEdgeInsetsZero;
+  CGFloat navigationBarHeight = CGRectGetHeight(self.navigationBar.frame);
+  if (@available(iOS 11.0, *)) {
+    currentSafeAreaInsets = self.view.safeAreaInsets;
+  }
+  UIEdgeInsets additionalSafeAreaInsets =
+      UIEdgeInsetsMake(0, 0, navigationBarHeight - currentSafeAreaInsets.bottom, 0);
+  if (@available(iOS 11.0, *)) {
+    self.selectedViewController.additionalSafeAreaInsets = additionalSafeAreaInsets;
+  } else {
+    self.content.layoutMargins = additionalSafeAreaInsets;
+    // Based on an article by Ryan Fox, attempt to adjust the contentInset of either the view
```

Comment:
I've typically not seen referencing of links, would you be able to explain here instead what's going on from your understanding. Personally, I'd prefer a TLDR than reading a full blog post

Reply:
The TLDR is this sentence. I'd be happy to add to the comment if you feel something is missing. The link is provided so anyone interested can get more information there.